### ISSUE TITLE
Various item states reset when exiting the Mines chase area

### ISSUE THREAD

**Issue by ZMBanshee** [UnderscoreKilburn/repentance-issues#11]
Examples include Bumbo, Damocles, Paschal Candle, Strawman, Keeper and Magdalene's Birthright, and others (will add more here).

**Comment by Lonrite** [comment]
yeah this applies to perfection and a bunch of other stuff i'm forgetting i think

**Comment by Rchardon** [comment]
Blood oath stat ups

**Comment by ez-duke** [comment]
Mom's kiss trinket gives empty containers
